### PR TITLE
avatar image and tooltip for bubbles

### PR DESCRIPTION
**Pull request by Priyami**
- Contributors git avatar image added inside the bubbles
        * `<def><pattern "id":"contributor_repo_id"><image "xlink:href": "avatar_url"></image></pattern></def>` with attributes defined.
        * For each node, `<circle "fill":"#avatar_url"></circle>`

- Tooltip added for each bubble
        * Tooltip` <div>` is defined with attributes like opacity, id, background color
        * Three function `showtooltip() movetooltip hidetooltip` and its attributes defined in each function
        * In each circle, event handler called as below.
   `.on("mouseover", showTooltip)
    .on("mousemove", moveTooltip)
    .on("mouseleave", hideTooltip)`

### CODE REVIEW

**Comment by Priyami** [comment]
@princiya 

#59 f5001172cd069f3ba63ceed9795d13a0d90cbf7c 

- Tooltip `<div>` was created, when I looked into the Developer tool. But it is not rendering on the screen. I have tried many attributes like z-index, background color, visibility. Nothing works. Please have a look.

**Review by Priyami — COMMENTED** [review]
Tooltip rendered on the DOM. Still, it's not perfectly pointed near the bubbles.

File: src/components/Resources/Bubblechart.js [hunk 1]
```diff
@@ -19,49 +19,83 @@ const Bubblechart = (aData) => {
...
+    .on("mousemove", mousemove)
+    .on("mouseout", mouseout);
+
+
+  let div = d3.select("#bubble").append("div")
+    .attr("class", "tooltip")
+    .style("display", "none");
+
+  function mouseover() {
+    div.style("display", "inline");
```
Comment (princiya) [inline src/components/Resources/Bubblechart.js:80]: `display: inline` may not behave correctly, given that the tooltip has width, height.
try using `display: inline-block` instead?

**Comment by Priyami** [comment]
https://jarrettmeyer.com/2018/06/21/d3-capturing-mouse-events 

-The above link shows how the mouse event captures the coordinates and other mouse events works.
- In our case using the mousemove event, we got the coordinates and the tooltip displayed on the top left corner of each circle.  We could add text or any link into the tooltip as required.

**Comment by princiya** [comment]
@Priyami there is a merge conflict with Bubblechart.css.
when this is resolved, we can merge this PR. Thank you for you work... :)

**Comment by Priyami** [comment]
> @Priyami there is a merge conflict with Bubblechart.css. when this is resolved, we can merge this PR. Thank you for you work... :)

@Princiya Thank you!. This is awesome and I like to learn and do more challenging work like this.

**Review by princiya — APPROVED** [review]

**Comment by princiya** [comment]
thanks @Priyami . good job 👏

**Pull request merged**